web/src/lib/components/onboarding-page/onboading-storage-template.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+  import { mdiCheck } from '@mdi/js';
+  import Button from '../elements/buttons/button.svelte';
+  import Icon from '../elements/icon.svelte';
+  import OnboardingCard from './onboarding-card.svelte';
+  import { createEventDispatcher, onMount } from 'svelte';
+  import { featureFlags } from '$lib/stores/server-config.store';
+  import StorageTemplateSettings from '../admin-page/settings/storage-template/storage-template-settings.svelte';
+  import { SystemConfigDto, api } from '@api';
+  import { user } from '$lib/stores/user.store';
+
+  const dispatch = createEventDispatcher<{
+    done: void;
+  }>();
+
+  let configs: SystemConfigDto | null = null;
+
+  onMount(async () => {
+    const { data } = await api.systemConfigApi.getConfig();
+    configs = data;
+  });
+</script>
+
+<OnboardingCard>
+  <p class="text-xl text-immich-primary dark:text-immich-dark-primary">STORAGE TEMPLATE</p>
+
+  <p>
+    The storage template is used to determine the folder structure and file name of your media files. You can use
+    variables to customize the template to your liking.
+  </p>
+
+  {#if configs && $user}
```

Comment:
What's the case that `$user` is falsy?

Reply:
`$user` are used in the `storage-template-settings`, so if it is not defined, the component will have issue. We will need to rework how `$user` work because there are cases where it has been undefined and the type is only `UserReponseDto` instead of `UserResponseDto | undefined`